tools/depends/configure.ac

```diff
@@ -83,6 +83,11 @@ AC_ARG_WITH([sdk],
   [specify sdk platform version (optional for android). default is android-21])],
   [use_sdk=$withval])
 
+AC_ARG_WITH([ndk],
```

Comment:
This changes behavior for an existing setting, should we go with api or ndk-version/level instead?